src/MainnetController.sol

```diff
@@ -751,6 +758,49 @@ contract MainnetController is AccessControl {
         }));
     }
 
+    // NOTE: !!! This function does not have integrations tests !!!
+    //       Set rate limit to zero.
+    function transferTokenLayerZero(
+        address oftAddress,
+        address target,
+        uint256 amount,
+        uint32  dstEid
+    )
+        external
+    {
+        _checkRole(RELAYER);
+        _rateLimited(
+            keccak256(abi.encode(LIMIT_LAYERZERO_TRANSFER, oftAddress, target, dstEid)),
+            amount
+        );
```

Comment:
Move this to a new mapping `layerZeroRecipients` and make the same function as `setMintRecipient` but for `setLayerZeroRecipient`

Reply:
added a new mapping